crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1122,15 +1122,27 @@ fn visit_node(
         }
     };
 
+    // Save the vector frame size which is needed to compare against the
+    // runtime frame size to calculate the scaling factor if a vector's
+    // constraints are set to scale.
+    let mut vector_size: Option<(f32, f32)> = None;
+    if let Some(size) = &node.size {
```

Comment:
I think you can collapse this --
```rust
if let Some(Size { x: Some(x), y: Some(y) } = &node.size {
...
```

Reply:
Ok fixed, plus rebased and added serialized files.